test/general-tests/constructor-test/duplicate_constructor.cairo

```diff
@@ -0,0 +1,26 @@
+#[contract]
+mod Contract {
+    struct Storage {
+        balance: felt252,
+    }
+
+    #[constructor]
+    fn constructor(initial_balance: felt252) {
+        balance::write(initial_balance);
+    }
+
+    #[constructor]
+    fn constructor2(initial_balance: felt252) {
+        balance::write(initial_balance);
+    }
+
+    #[external]
+    fn increase_balance(amount1: felt252, amount2: felt252) {
```

Comment:
Here and in other new files, can we remove these unused external/view methods? Only constructors are needed, right? Maybe even remove the storage and initial_balance being a parameter of the constructor.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/377/commits/9b6048abf937d5c0936f1f5a4bf065acd4449935